lib/const.rb

```diff
@@ -36,10 +36,13 @@
 end
 
 if ENV['CREW_CACHE_DIR'].to_s == ''
-  CREW_CACHE_DIR = HOME + '/.cache/crewcache/'
+  CREW_CACHE_DIR = HOME + '/.cache/crewcache'
 else
   CREW_CACHE_DIR = ENV['CREW_CACHE_DIR']
 end
+FileUtils.mkdir_p CREW_CACHE_DIR unless Dir.exist? CREW_CACHE_DIR
+
+CREW_IS_CACHING = ENV['CREW_IS_CACHING']
```

Comment:
I'm thinking a better name would be `CREW_CACHE_ENABLED`.

Reply:
I like that, yes. Lemme make that change.